plugins/modules/win_certificate_store.ps1

```diff
@@ -229,12 +229,25 @@ Function New-CertFile($module, $cert, $path, $type, $password) {
         "pkcs12" { [System.Security.Cryptography.X509Certificates.X509ContentType]::Pkcs12 }
     }
     if ($type -eq "pkcs12") {
-        $missing_key = $false
-        if ($null -eq $cert.PrivateKey) {
-            $missing_key = $true
-        }
-        elseif ($cert.PrivateKey.CspKeyContainerInfo.Exportable -eq $false) {
-            $missing_key = $true
+        $missing_key = $true
+        if ($cert.HasPrivateKey) {
+            switch ($cert.PublicKey.Oid.FriendlyName) {
```

Comment:
Is there any benefit to actually trying to export the private key bytes vs just relying on `HasPrivateKey`? I see you are checking the `ExportPolicy` later on but is that needed at all vs just relying on `Export()` to give us what we need and potentially failing?

Avoiding having to deal with the different key types would be nice plus these methods require the host to have dotnet framework 4.6 which isn't a current guarantee. If we do need these checks we should only optionally do them if the type/method is present in the runtime.